src/components/Home/home.css

```diff
@@ -0,0 +1,119 @@
+.home-container {
+    min-height: 70vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+
+.banner-container {
+    height: 40rem;
+    width: 100%;
+    padding: 2rem 2rem 0 2rem;
+}
+
+.banner {
+    position: relative;
+    height: 100%;
+    width: 100%;
+}
+
+.banner-image img {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.banner-content-container {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+}
+
+.banner-content {
+    height: 100%;
+    width: max-content;
+    position: absolute;
+    top: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    color: var(--secondary-white-text-color);
+    background-color: var(--light-black-background);
+    opacity: 0.8;
+    padding: 2rem;
+    border-radius: 10px;
+}
+
+.banner-heading {
+    font-size: 4rem;
+}
+
+.banner-sub-heading {
+    font-size: 3.2rem;
+}
+
+.arrow-down {
+    font-size: 3rem;
+}
+
+.banner-btn {
+    padding: 1rem 0.5rem;
+    font-size: 2rem;
+    background-color: #09cc02;
+    border-radius: 5px;
+}
+
+.banner-btn:hover {
+    background-color: #19a514;
```

Comment:
here as well

Reply:
Thank you for pointing out...Thank you for PR review